azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -309,19 +306,6 @@ internal open class CallCompositeActivity : AppCompatActivity() {
         }
     }
 
-    private fun configureActionBar() {
-        supportActionBar?.setDisplayHomeAsUpEnabled(true)
```

Comment:
Do we need to set visibility to force false? Mean, can Contoso turn on visibility for app wide and every activity needs to set false if required?

Reply:
No. This would result in a crash since we are using No Action Bar theme not trying to use the default action bar APIs.